app/views/external_users/admin/external_users/_form.html.haml

```diff
@@ -1,62 +1,50 @@
-        %legend#radio-control-legend-email
-          %span.heading-medium
-            = t('.email_notification')
-
-        = user_fields.collection_radio_buttons(:email_notification_of_message, [true, false], :to_s, :to_s) do |b|
-          .multiple-choice
-            = b.radio_button('aria-labelledby': "radio-control-legend-email #{b.text.to_css_class}")
-            = b.label { (b.value.to_s == 'true' ? t('.answer_yes') : t('.answer_no')) }
+      = user_fields.govuk_email_field :email_confirmation,
+        label: { text: t('.email_confirmation') }
+
+    = f.govuk_collection_radio_buttons :email_notification_of_message,
+      [['Yes','true'],['No','false']],
+      :last,
+      :first,
+      inline: true,
+      legend: { text: t('.email_notification'), size: 's' }
 
     - if current_user.persona.admin?
-      .form-group
-        %fieldset.inline.js-advocate-roles{ 'aria-describedby': 'checkbox-control-legend-roles' }
-          %legend#checkbox-control-legend-roles
-            %span.heading-medium
-              = t('.roles')
-
-          = f.collection_check_boxes(:roles, @external_user.available_roles, :to_s, :to_s) do |b|
-            .multiple-choice
-              = b.check_box('aria-labelledby': "checkbox-control-legend-roles #{b.text.to_css_class}")
-              = b.label(id: b.text.to_css_class) { b.value.humanize }
-
-    - if current_user.persona.provider.chamber?
-      .form-group.js-chamber-advocates-only
-        %fieldset.inline{ 'aria-describedby': 'checkbox-control-legend-vat' }
-          %legend#checkbox-control-legend-vat
-            = t('.vat_registered')
-
-          = f.collection_radio_buttons(:vat_registered, [true, false], :to_s, :to_s) do |b|
-            .multiple-choice
-              = b.radio_button('aria-labelledby': "checkbox-control-legend-vat #{b.text.to_css_class}")
-              = b.label { (b.value.to_s == 'true' ? t('.answer_yes') : t('.answer_no')) }
-
-      .form-group
-        = f.label :supplier_number, t('.supplier_number'), class: 'form-label-bold'
-        = f.text_field :supplier_number, class: 'form-control'
-
-  .form-group
-    = govuk_button(t('.save'))
+      = f.govuk_check_boxes_fieldset :roles, legend: { text: t('.roles'), size: 's' } do
+
+        = f.govuk_check_box :roles,
+          @external_user.available_roles.first,
+          label: { text: 'Admin' },
+          link_errors: true
+
+        = f.govuk_check_box :roles,
+          @external_user.available_roles.last,
+          label: { text: 'Advocate' } do
+
+          - if current_user.persona.provider.chamber?
+            = f.govuk_collection_radio_buttons :vat_registered,
+              [['Yes','true'],['No','false']],
+              :last,
+              :first,
+              inline: true,
+              legend: { text: t('.vat_registered'), size: 's' }
+
+    = f.govuk_text_field :supplier_number,
+      hint: { text: t('.supplier_number_hint') },
```

Comment:
do we need to worry about the safari bug here i.e. adding hints/`aria-describedby` impact on buttons?

Reply:
Not here @jsugarman as this radio collection has no hint.